src/components/structures/RoomView.js

```diff
@@ -1052,13 +1061,19 @@ module.exports = React.createClass({
                     );                
                 }
                 else {
+                    var inviterName = undefined;
+                    if (this.props.oobData) {
+                        inviterName = this.props.oobData.inviterName;
+                    }
+
                     return (
                         <div className="mx_RoomView">
-                            <RoomHeader ref="header" room={this.state.room} simpleHeader="Join room"/>
```

Comment:
It's referenced by the Create Room screen which is no longer hooked up to anything, and the user settings screen (which annoys me because settings is not a room).